JSON.md

```diff
@@ -172,3 +172,390 @@ cfbs init && cfbs add https://github.com/cfengine/some-repo
   ]
 }
 ```
+
+
+## Modules with input
+Some modules allow for users to add module input by responding to questions
+expressed under the `"input"` attribute in `cfbs.json`. User input can be added
+using the `cfbs input <module-name>` command, which stores responses in
+`./<module-name>/input.json`. These responses are translated into augments which
+will be added to `./out/masterfiles/def.json` during `cfbs build`.
+
+### Create single file example:
+The `"input"` attribute takes a list of input definitions as illustrated below.
+The value of the `"input"` attribute may be expressed as inline JSON, or as a
+file path or URL to an external JSON file.
```

Comment:
Do we have a real reason/use-case for inputs specified in an external JSON file? If not, let's not complicate our lives by allowing this, at least not for now.